### PR TITLE
Pushing script

### PR DESCRIPTION
**Pull request by Priya-kunwar24**
Pushing script Create_claim_without_login for both brands and some xpath path and shared test cases changes.

### CODE REVIEW

File: src/test/java/ApplicationPages/ContactInformationPage.java [hunk 1]
```diff
@@ -8,18 +8,22 @@
 	@FindBy(id = "email")
 	public WebElement textbox_EmailID;
 	
-	@FindBy(id = "mat-input-7")
+	//@FindBy(id = "mat-input-7")
```
Comment (bhawesh11) [inline src/test/java/ApplicationPages/ContactInformationPage.java:11]: unused comment should be removed from code

File: src/test/java/ApplicationPages/TellUsMore.java [hunk 2]
```diff
@@ -44,7 +44,8 @@
 	@FindBy(id="city")
     public WebElement textbox_City;
 	
-	@FindBy(xpath="//mat-label[contains(text(),'State')]//ancestor::label")
+	//@FindBy(xpath="//mat-label[contains(text(),'State')]//ancestor::label")
```
Comment (bhawesh11) [inline src/test/java/ApplicationPages/TellUsMore.java:47]: Remove commented/unused code

File: src/test/java/testApparent/TC_Apparent.java [hunk 3]
```diff
@@ -27,15 +27,26 @@ public void openBrowser() throws IOException {
...
 	STC_WitnessPolice stc_WitnessPolice = new STC_WitnessPolice();
+	STC_TellUsMore stc_tellUsMore = new STC_TellUsMore();
 
 	
 	// ==============================================================================================
 //	***_TEST_CASES_***
 
 
 	// TC001
-	@Test(enabled = true, priority = 5, description = "Simple Claim")
```
Comment (bhawesh11) [inline src/test/java/testApparent/TC_Apparent.java:38]: Enable test cases

**Review by bhawesh11 — CHANGES_REQUESTED** [review]

File: src/test/java/ApplicationPages/ContactInformationPage.java (hunk 1 above)
Comment (yogesh25) [reply src/test/java/ApplicationPages/ContactInformationPage.java]: resolved

Comment (yogesh25) [reply src/test/java/ApplicationPages/ContactInformationPage.java]: resolved

File: src/test/java/ApplicationPages/TellUsMore.java (hunk 2 above)
Comment (yogesh25) [reply src/test/java/ApplicationPages/TellUsMore.java]: resolved.

File: src/test/java/testApparent/TC_Apparent.java (hunk 3 above)
Comment (yogesh25) [reply src/test/java/testApparent/TC_Apparent.java]: resolved.

**Review by bhawesh11 — APPROVED** [review]

**Pull request merged**